spyder/plugins/console/widgets/main_widget.py

```diff
@@ -0,0 +1,522 @@
+# -*- coding: utf-8 -*-

```

Comment:
Please change the line ending of this file to Linux so we can add suggestions to it in the future.

Reply:
Fixed